mods/tnt/init.lua

```diff
@@ -27,12 +21,20 @@ minetest.register_on_mods_loaded(function()
 		cid_data[minetest.get_content_id(name)] = {
 			name = name,
 			drops = def.drops,
-			flammable = def.groups.flammable,
+			flammable = def.groups and (def.groups.flammable or 0) ~= 0,
```

Comment:
could also do this :man_shrugging: 
```suggestion
			flammable = core.get_item_group(name, "flammable") ~= 0,
```